hilti/toolchain/src/compiler/cxx/elements.cc

```diff
@@ -342,7 +342,9 @@ std::string cxx::type::Struct::str() const {
 
     auto fmt_member = [&](const auto& f) {
         if ( auto x = std::get_if<declaration::Local>(&f) ) {
-            if ( ! (x->isInternal() || x->linkage == "inline static") ) // Don't visit internal or static fields.
+            if ( x->isAnonymous() )
+                visitor_calls.emplace_back(fmt("_(\"<anon>\", %s); ", x->id));
```

Comment:
Nit: This is still slightly weird in that it seems to only apply to anon bitfield fields, so if users mix these switch bitfield constructs and other anon fields the output might given them pause.

I don't have a better suggestion and this is already pretty contrived, so totally fine if we just go with this for now.